test/unit_tests/unittest_scan_range.cpp

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) 2020 Pilz GmbH & Co. KG
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+#include <stdexcept>
+
+#include <gtest/gtest.h>
+
+#include "psen_scan_v2/scan_range.h"
+
+using namespace psen_scan_v2;
+
+namespace psen_scan_v2_test
+{
+static constexpr TenthOfDegree MIN_ALLOWED_ANGLE{ 7 };
+static constexpr TenthOfDegree MAX_ALLOWED_ANGLE{ 54 };
+
+static constexpr TenthOfDegree TOO_SMALL_SCAN_ANGLE{ MIN_ALLOWED_ANGLE.value() - 1u };
+static constexpr TenthOfDegree TOO_LARGE_SCAN_ANGLE{ MAX_ALLOWED_ANGLE.value() + 1u };
+
+using TestScanRange = ScanRange<MIN_ALLOWED_ANGLE.value(), MAX_ALLOWED_ANGLE.value()>;
+
+class ScanRangeBuilder
```

Comment:
Maybe this is a little bit over the top? How about just using the CTor in the Tests?

Reply:
Done.